### PR TITLE
fix: upload presentation only after hackathon start

### PR DESCRIPTION
**Pull request by reyhanrumengan**

### CODE REVIEW

**Comment by reyhanrumengan** [comment]
@LetsPoke currently i didn't made the changes for the myIdeas page, because I thought we discussed that we want to remove this page. What are your thought? Should I for now also implement this in the myIdeas page?

File: hms-app/src/components/buttons/CardButton.tsx [hunk 1]
```diff
@@ -122,6 +124,13 @@ export default function CardButtons(props: IProps) {
     </Modal>
   )
 
+  const uploadButton = () => {
+    return (
+      type === IdeaCardType.AllIdeas &&
```
Comment (LetsPoke) [inline hms-app/src/components/buttons/CardButton.tsx:129]: do we really need that check? 
I would say we would need to check if its the admin page or not
on all pages it should only be visible when hackathon started, excpect for the admin page there it should be always visible

**Comment by LetsPoke** [comment]
> @LetsPoke currently i didn't made the changes for the myIdeas page, because I thought we discussed that we want to remove this page. What are your thought? Should I for now also implement this in the myIdeas page?

as it should be not much of work I would also add it to myideas and ideaPool

File: hms-app/src/components/buttons/CardButton.tsx (hunk 1 above)
Comment (reyhanrumengan) [reply hms-app/src/components/buttons/CardButton.tsx:129]: that does makes sense. I forgot about the admin page

**Review by LetsPoke — APPROVED** [review]

**Pull request merged**